api/validation_models.py

```
@@ -1,17 +1,59 @@
 from datetime import datetime
 from enum import Enum
 import re
-from typing import Any
+import sys
```

Flake8 (7.3.0)

[error] 'sys' imported but unused

(F401)